### ISSUE TITLE
Test case ISO19115

### ISSUE THREAD

**Issue by fleimgruber** [renalreg/PyXB-X#17]
For my use case and maybe valuable as a test case for PyXB-X I tried code generation of bindings to https://github.com/ISO-TC211/XML.

This is the command I tried:
```
python scripts/pyxbgen -m x -u "file:///c:/Users/LeimgruberF/dev/XML/schemas.isotc211.org/19115/-3/mds/2.0/mds.xsd"
```
which errors with:
```
Exception generating bindings: Target file .\_gco.py for module MR[urn:uuid:d0d82af2-e9f6-442d-98ad-7fb54ef1612a]@http://www.isotc211.org/2005/gco bindings exists with other content
Traceback (most recent call last):
  File "C:\Users\LeimgruberF\dev\PyXB-X\pyxb\binding\generate.py", line 1946, in modulePathData
    binding_file = pyxb.utils.utility.OpenOrCreate(binding_file_path, tag=module.moduleUID())
  File "C:\Users\LeimgruberF\dev\PyXB-X\pyxb\utils\utility.py", line 826, in OpenOrCreate
    raise OSError(errno.EEXIST, os.strerror(errno.EEXIST))
FileExistsError: [Errno 17] File exists

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\LeimgruberF\dev\PyXB-X\scripts\pyxbgen", line 65, in <module>
    modules = generator.bindingModules()
  File "C:\Users\LeimgruberF\dev\PyXB-X\pyxb\binding\generate.py", line 2987, in bindingModules
    self.__generateBindings()
  File "C:\Users\LeimgruberF\dev\PyXB-X\pyxb\binding\generate.py", line 2926, in __generateBindings
    nsm = NamespaceModule(self, mr, mr_scc)
  File "C:\Users\LeimgruberF\dev\PyXB-X\pyxb\binding\generate.py", line 1700, in __init__
    self._setModulePath(generator.modulePathData(self))
  File "C:\Users\LeimgruberF\dev\PyXB-X\pyxb\binding\generate.py", line 1949, in modulePathData
    raise pyxb.BindingGenerationError('Target file %s for module %s bindings exists with other content' % (binding_file_path, mr))
pyxb.exceptions_.BindingGenerationError: Target file .\_gco.py for module MR[urn:uuid:d0d82af2-e9f6-442d-98ad-7fb54ef1612a]@http://www.isotc211.org/2005/gco bindings exists with other content
```
So the question I have is whether the problem lies in:

- the code generation
- the upstream .xsd

**Comment by jtc42** [comment]
I think whatever the cause there's a good chance this would be considered out-of-scope for the library at this point. This fork really only exists to keep the last release of PyXB working on later releases of Python, essentially freezing functionality to ensure there's no chance of breaking compatibility. If it turns out the fix for this is trivial, and guaranteed not to break anything, we can merge it in, but if it requires substantial code changes then we likely won't.

I'd generally suggest alternatives like XSData at this point (other options might be better, I've not looked in a while).

**Comment by fleimgruber** [comment]
Thanks for checking back and the explanation.

I was using xsdata in parallel already and recently got unblocked there, so I will continue with xsdata.
